src/cmdlib.sh

```diff
@@ -267,6 +267,10 @@ commit_overlay() {
     name=$1
     path=$2
     respath=$(realpath "${path}")
+    # Only keep write bit for owner for all overlay files/dirs. This modifies
+    # the source config, but meh... git doesn't track it anyway and it's easier
+    # than using ostree's --statoverride when dealing with executable files.
```

Comment:
At some point we may actually want to mount in the source config read-only or so.

I think we can try to get through some sort of CLI flag like...`ostree commit --mode=uo-w` or so.  But fine for now.

Reply:
Yeah, agreed that'd be nice to have. I think also we should support `-<mode>` in stat override files given that we support `+`. (I was debating adding that actually for this... but it just didn't seem worth it.)

Reply:
> Yeah, agreed that'd be nice to have. I think also we should support `-<mode>` in stat override files given that we support `+`. (I was debating adding that actually for this... but it just didn't seem worth it.)

maybe make an issue and reference it here?